amclient/amclient.py

```diff
@@ -596,16 +599,38 @@ def create_package(self):
             method=utils.METHOD_POST,
         )
 
+    def extract_file_stream(self):
+        """Extract a file, relative to an AIP's path. The primary functionality
+        is provided by extract_file below. This helper command exists to change
+        the response behavior specifically for the command-line i.e. to enable
+        the output of a stream's content directly to stdout.
+        """
+        self.stream = True
+        self.cli_call = True
```

Comment:
Should we add this two to the list of parameters [here](https://github.com/artefactual-labs/amclient/blob/d81101451a9a65fb1ab780b6a98f2d9fa9ae05d1/amclient/amclient.py#L107-L118)? One could initiate AMClient with them and get different behaviors calling `extract_file`.